### PR TITLE
feature/user course management chrisg simone day 2

### PR DESCRIPTION
**Pull request by BUGBOUNTYchrisg8691**
# What's here (& why)

- (include link(s) to applicable Trello card(s))

# Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete / work-in-progress, PR is for discussion or feedback
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How has this been tested?

- [x] Tested locally
- [ ] Test suite provides the same results as prior to making changes

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/RestUsers/EditUserForm.js [hunk 1]
```diff
@@ -36,13 +78,17 @@ const EditUserForm = props => {
       userrole: values.role,
     };
 
-    client.patchUser(editedUser.userid, editedUser);
+    // client.patchUser(editedUser.userid, editedUser);
```
Comment (chazkiker2) [inline src/components/RestUsers/EditUserForm.js]: delete commented-out code

File: src/components/common/CourseEnrollmentCheckbox.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  line-height: inherit;
+}
+`
+
+const CourseEnrollmentCheckbox = props => {
+  const [checked, setChecked] = React.useState(props.value);
+
+  const onChange = e => {
+    setChecked(!checked);
+    // if (checked) {
```
Comment (chazkiker2) [inline src/components/common/CourseEnrollmentCheckbox.js]: delete commented-out code

**Review by simonesquad — APPROVED** [review]
looks good to me

**Review by chazkiker2 — CHANGES_REQUESTED** [review]
Ready to merge after deleting commented-out code

**Review by sebass482 — APPROVED** [review]
Tested locally and looks good 👌

**Review by chazkiker2 — APPROVED** [review]

**Pull request merged**